config/routes.rb

```diff
@@ -27,24 +29,28 @@
     resources :supplementals, only: %w[index], module: 'applications'
   end
 
-  resources :pattern_library, only: %w[index]
-
-  ## --- API namespacing
+  # API namespacing
   namespace :api do
     namespace :v1 do
-      scope '/short-form' do
-        post 'submit' => 'short_form#submit'
-      end
-      scope '/flagged-applications' do
-        put 'update' => 'flagged_applications#update'
-      end
+      get 'ami' => 'ami#get'
 
       resources :applications, only: %w[index update]
+
       scope '/field-update-comments' do
         post 'create' => 'field_update_comments#create'
       end
-      get 'ami' => 'ami#get'
+
+      scope '/flagged-applications' do
+        put 'update' => 'flagged_applications#update'
+      end
+
       resources :preferences, only: %w[update]
+
+      resources :rental_assistances, path: '/rental-assistances', only: %w[create update destroy]
```

Comment:
This is the only important changed line relevant to rental assistances in this file. The rest of the changes are just tidying and alphabetization.